app/views/accounts/_account_list.html.erb

```diff
@@ -0,0 +1,32 @@
+<%# locals: (type:) -%>
+
+<% accounts = Current.family.accounts.where(accountable_type: type.name) %>
+
+<div class="text-sm mb-1" data-controller="reveal">
+  <button class="flex gap-4 px-2 py-3 items-center w-full rounded-[10px] font-medium hover:bg-[#f2f2f2]" data-action="reveal#toggle">
```

Comment:
@JoshAntBrown any thoughts on using summary/detail element for this?

Reply:
Good call, will refactor, wanted to get a first version out

Reply:
Much better with the detail element, no need for the stimulus controller, and toggling between the icons feels more robust being based on the state of the dialog. Thanks!